docs/EN/operate/build.md

```diff
@@ -16,22 +16,43 @@ The Trio repository contains instructions for building the Trio app using a brow
 :::
 
 * Fork from: [https://github.com/nightscout/Trio](https://github.com/nightscout/Trio)
-* `Identifier Names` will be: `FreeAPS`, `FreeAPSWatch`, `FreeAPSWatch WatchKit Extension`, `LiveActivityExtension`
-    * These names are the same as that used by iAPS, they are distinguised by the Identifier itself, which include the new BundleID for your Trio app
-    * If you build with Xcode before, they will start with XC and use the Identifier string rather than the Name
-* `Identifiers` will be:
-    * `org.nightscout.TEAMID.trio`
-    * `org.nightscout.TEAMID.trio.watchkitapp`
-    * `org.nightscout.TEAMID.trio.watchkitapp.watchkitextension`
-    * `org.nightscout.TEAMID.trio.LiveActivity`
-* Only the first 3 of these identifiers need to have the `App Group` added
-* For the list below, make sure the box next to `App Group` is checked, and click `Configure` to select the same app group as Loop: `group.com.TEAMID.loopkit.LoopGroup`
-* You must add this `App Group` to each of the following identifiers:
-    * `org.nightscout.TEAMID.trio`
-    * `org.nightscout.TEAMID.trio.watchkitapp`
-    * `org.nightscout.TEAMID.trio.watchkitapp.watchkitextension`
+* Identifiers for Trio: see [Table of Identifiers](#table-of-identifiers)
+* You use the `Loop App Group` for Trio, see [Create the `Loop App Group`](#create-the-loop-app-group)
+* Add the `Loop App Group` to these identifiers:
+    * Trio
+    * Trio Watch
+    * Trio WatchKit Extension
 * In `App Store Connect`, the `Bundle ID` for Trio will be: `org.nightscout.TEAMID.trio`
 
+### Create the `Loop App Group`
+
+If you already have a `Loop App Group`
+
+* You can skip this step - your existing App Groups are found at this link: [App Group List](https://developer.apple.com/account/resources/identifiers/list/applicationGroup)
+* If your `Loop App Group` was created from a Mac with Xcode, you may choose to edit the Description to make the **NAME** match
+
+If you do not have a `Loop App Group`:
+
+* Go to [Register an App Group](https://developer.apple.com/account/resources/identifiers/applicationGroup/add/) on the apple developer site and use the table below to help you create one.
+
+| NAME | XCode version | IDENTIFIER |
+|:--|:--|:--|
+| Loop App Group | group com TEAMID loopkit LoopGroup| group.com.7PT47XGDN4.loopkit.LoopGroup |
```

Comment:
Should we use TEAMID instead of 7PT47XGDN4 here?

Reply:
Absolutely. Thanks for catching that.